src/ms-review.tex

```diff
@@ -1879,24 +1887,18 @@ \section{Summary and Outlook}
 
 \begin{acknowledgements}
 
-	Mention Christoph here?
-
 	We would like to thank the \texttt{Numpy}, \texttt{Scipy}, \texttt{IPython} and
 	\texttt{Matplotlib} communities for providing their packages which are
-	invaluable to the development of Astropy. We thank the \github team for
+	invaluable to the development of Gammapy. We thank the \github team for
 	providing us with an excellent free development platform. We also are grateful
 	to Read the Docs (\ReadthedocsUrl), and Travis (\TravisUrl) for providing free
```

Comment:
do we still use read the docs?

Reply:
No, but we used it for a long time, as well as Travis so probably fine to keep...